client/src/components/clan/ClanRouter.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Forum from './Forum.jsx';
+import About from './About.jsx';
+import Events from './Events.jsx';
+import Members from './Members.jsx';
+import NewClan from './NewClan.jsx';
+import ClanSearch from './ClanSearch.jsx';
+import { Switch, Route, Link } from 'react-router-dom';
+import { Media } from 'react-bootstrap';
+
+/**
+ * Class representing the React Clan Container Component.
+ * @extends Clan
+ */
+
+class ClanRouter extends React.Component {
+  constructor (props) {
+    super(props);
+  }
+
+  addNewClan(clan) {
+    console.log('Added New Clan', clan);
+    // TODO: post to database
```

Comment:
I could set up this endpoint later tonight, I would need to iron out the clan crud methods first. Could be a goal for standup tonight?